protocol/contracts/TrucoMatchFactory.sol

```diff
@@ -81,6 +82,6 @@ contract TrucoMatchFactory is OwnableUpgradeable {
 
     // Destroy match contract
     function destroyMatch(address _matchAddress) public onlyOwner {
-        // Destroy match
+        // TODO: Destroy match
```

Comment:
i think this function has no purpose since match destruction logic should be within match